app/src/main/java/com/example/pygmyhippo/database/StorageOnCompleteListener.java

```diff
@@ -0,0 +1,59 @@
+package com.example.pygmyhippo.database;
+
+import androidx.annotation.NonNull;
+
```

Comment:
Does this file have a header comment? We should make sure to add that as a PR requirement because it is difficult to go back and comment everything

Reply:
Or wait I guess I see it below. Although I think we need that Javadoc and a normal header comment and that's where you list the issues and purposes. It looks like this comment was reused anyways. Maybe you should specifiy what this listener is for.

Reply:
makes sense I'll do that